oxli/common.py

```diff
@@ -0,0 +1,172 @@
+#
```

Comment:
I had to change a few of the `build_parser` funcs to add to a parser instead of generate one (because of how subparsers work).

I could modify the khmer ones to be along the line of
```
build_hashbits(parser=None):
   if parser == None:

```